### DISCUSSION TITLE
PROPOSAL: The Second CRO Community Burn on Cronos POS Chain

### DISCUSSION THREAD

**Discussion by lezzokafka** [crypto-org-chain/chain-main#1050] [Ideas]
![Burn V1](https://github.com/crypto-org-chain/chain-main/assets/51898510/84ffc176-ec40-467f-beb0-32afb0a00f0c)
# Introduction
Following the first successful burn of 50 Million $CRO, we are proposing to initiate the second burn of 50 Million of $CRO on the Cronos POS Chain. Once this proposal is passed, an additional 50 Million of $CRO will be burned -  **Leading to a total of 100 Million $CRO permanently removed from the circulating supply.**

## Background
As introduced in the previous proposal, the "Burning Mechanism of $CRO" has been implemented and has been continuously allocating a portion of the $CRO generated through emissions to the community pool at the Cronos POS Chain protocol level. The accumulated amount is then periodically burned to complete the cycle. At the time of writing, the community pool holds approximately [61 Million](https://burninfo.cronos-pos.org/) of $CRO and this proposal suggests a burning of 50 Million from the amount.

# Technical Details
To execute the "burn”, a "community pool spend" governance proposal with a target address needs to be initiated. In particular, funds from the community pool are proposed to be moved to the "dead" address (i.e.[cro1qqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqtcgxmv](https://cronos-pos.org/explorer/account/cro1qqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqtcgxmv)), which no one has control over, and effectively removes these funds from the circulating supply permanently. The current pending and burned amount, as well as historical burn events, are on the [burn info page](https://burninfo.cronos-pos.org/). Moreover, we can use this [endpoint](https://cronos-pos.org/explorer/api/v1/supply/circulating) to query the current value of the circulating supply at any time, note this number is provided in `basecro`, where `1 CRO = 10^8 basecro` on the Cronos POS chain.

# Governance Votes
After discussion and suggestion to this proposal, an on-chain network upgrade governance proposal will be raised. The following items summarize the voting options and what they mean for this proposal.
- YES: You approve of the proposal on the changes to proposed changes to move 50 Million (`50,000,000`) of $CRO from the community pool to the “dead” address and remove this amount from the circulating supply of CRO.
- NO: You disapprove of the proposal in its current form. Feel free to give feedback and suggestions on how to improve the proposal.
- NO (VETO): You veto the entire motivation for the proposal and are strongly opposed to its implementation.
- ABSTAIN: You are impartial to the outcome of the proposal. If the quorum is not met or less than 1/3 of the participating voting power has voted No, the proposal fails and is considered to be vetoed. In this case, the proposal deposit is burned rather than being returned to respective depositors.

## Let us know your thoughts
The importance of active community participation in shaping the future of the Cronos POS chain cannot be overstated. Engage with your delegated validators and participate in proposal discussions on Discord and GitHub. Your input is vital in fostering a vibrant that benefits all $CRO holders and the wider ecosystem.

**Comment by alan-killie** [comment]
The reason for the pool was total supply concerns from the community. I don't understand how we need a vote to decide what to do with the pool every 6 months or 50 million tokens. It was implemented to reduce supply. Didn't the initial proposal already discuss time based or amount based burns?

https://blog.cronos-pos.org/p/cro-community-burn-proposal-explainer

**Reply by 87x23** [reply]
Definitely should be an automatic burn… considering it would take 10 years to burn 1billion at 50million every six months.

**Reply by zugsoft** [reply]
The proposal was to automatic burn CRO when the community pool hits the 50M CRO threshold.
